cli/src/commands/auth/commands/login.ts

```diff
@@ -45,16 +48,51 @@ export default (opts: BaseCommandOptions) => {
       program.error('Could not perform authentication. Please try again');
     }
 
-    const organizations = new Set(decoded.groups.map((group) => group.split('/')[1]));
+    const organizationSlugs = new Set(decoded.groups.map((group) => group.split('/')[1]));
+
+    const organizationSlugByDisplayKey = new Map<string, string>();
+    const organizationKeys: Array<string> = [];
+    const spinner = ora('Fetching organizations...').start();
+    for (const organizationSlug of organizationSlugs) {
+      const headers = getBaseHeaders();
+      const response = await opts.client.platform.getOrganizationBySlug(
```

Comment:
Please use Promise.all() instead of synchronous

Reply:
not convenient for the fallback